data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -153,6 +153,40 @@
     "Percent of individuals < 100% Federal Poverty Line in 2010"
 )
 
+# RSEI Aggregated Micro-data
+EPA_RSEI_NUMBER_FACILITIES_OUTPUT_FIELD = (
+    "Number of facilities affecting the tract"
+)
+EPA_RSEI_NUMBER_RELEASES_OUTPUT_FIELD = "Number of releases affecting the tract"
+EPA_RSEI_NUMBER_CHEMICALS_OUTPUT_FIELD = (
+    "Number of chemicals affecting the tract"
+)
+EPA_RSEI_AVERAGE_TOXICITY_OUTPUT_FIELD = (
+    "Average toxicity-weighted concentration of the cells in the tract"
+)
+EPA_RSEI_SCORE_OUTPUT_FIELD = (
+    "Sum of the risk-related scores of the cells in the tract"
+)
+EPA_RSEI_CSCORE_OUTPUT_FIELD = "Sum of the risk-related scores of the cells in the tract (Cancer toxicity weights)"
+EPA_RSEI_NCSCORE_OUTPUT_FIELD = "Sum of the risk-related scores of the cells in the tract (Noncancer toxicity weights)"
+EPA_RSEI_POPULATION_OUTPUT_FIELD = (
+    "Sum of the population of the cells in the tract"
+)
+
+EPA_RSEI_SCORE_PERCENTILE_LABEL_FIELD = (
+    "Percentile intervals for overall risk scores"
+)
+EPA_RSEI_CSCORE_PERCENTILE_LABEL_FIELD = (
+    "Percentile intervals for other risk scores (Cancer toxicity weights)"
+)
+EPA_RSEI_NCSCORE_PERCENTILE_LABEL_FIELD = (
+    "Percentile intervals for other risk scores (Noncancer toxicity weights)"
+)
```

Comment:
These columns reference bins for percentiles. I am wondering if the word choice is appropriate?

Reply:
Yeah I'm not sure. See my comment above about whether these are necessary?